src/atoms/link/Link.stories.js

```diff
@@ -1,33 +1,46 @@
-import { select, text } from '@storybook/addon-knobs'
-
-import getClassKnobsConfig from '@utils/helpers/get-class-knobs-config.js'
-import selectorsConfig from './Link.selectors.json'
-
 import ALink from './Link.vue'
 
-const classKnobsConfig = getClassKnobsConfig(selectorsConfig)
-
 export default {
   title: 'Atoms/Link',
-  component: ALink
+  component: ALink,
+  argTypes: {
+    tag: {
+      control: {
+        type: 'text'
+      }
+    },
```

Comment:
is this needed? this control seems to generate without any additional configuration